views/getting_started.dt

```diff
@@ -10,6 +10,24 @@ block body
 
 	p To install DUB, search your operating system's package manager or <a href="https://github.com/dlang/dub/releases">download</a> the pre-compiled package for your platform. The Windows installer will perform all installation steps; for other archives, you will want to ensure the DUB executable is in your path. Installation from source on other platforms is as simple as installing the dmd development files and your system's libcurl-dev, then running <code>./build.sh</code> in the repository's folder.
 
+	h2#running Running DUB
+
+	p Some common commands:
+
+	ul
+		li #[code dub build] - build the project in the current folder
+		li #[code dub run] - build & run the project in the current folder (default)
+		li #[code dub run package] - build & run the latest version of `package`
+		li #[code dub fetch package] - fetch the latest version of `package`
+
+	p Other versions of `package` can be specified:
+
+	ul
+		li #[code package@1.0] - the 1.0 version of `package`
+		li #[code package@~master] - the latest git `master` version of `package`
```

Comment:
the branch versioning is considered deprecated - also I don't really think it's that necessary to know either of these two advanced versioning syntaxes in the overview page. The documentation of the commands should help with that.

Also you may be interested in https://docs.webfreak.org, which is GitHub on https://github.com/WebFreak001/dub-docs-v2, which has CLI documentation as well as more extended documentation pages and is all written in markdown.

Reply:
Oh great, thanks I'll take a look.

Reply:
> the branch versioning is considered deprecated

When I use `--version`, dub says that's deprecated:
```sh
$ dub --version
DUB version 1.33.0, built on Jun  2 2023
$ dub fetch diet-ng --version 2.0.0
     Warning The '--version' parameter was deprecated, use diet-ng@2.0.0. Please update your scripts.
```
What should I be using?

> I don't really think it's that necessary to know either of these two advanced versioning syntaxes in the overview page. The documentation of the commands should help with that.

OK, but where is that documented? I can't find it. Occasionally it mentions one, but it doesn't seem to explain it, e.g. does a leading `~` mean git branch?

Reply:
OK I have found an issue, it seems it's not documented:
https://github.com/WebFreak001/dub-docs-v2/issues/12